templates/donarInfo.hbs

```diff
@@ -0,0 +1,27 @@
+{{!-- Donation info modal --}}
+<section class="modal">
+        <div class="modal__background"></div>
+        <div class="modal__content">
+            <div class="modal__image">
+              <img src="{{assetSrc image}}" alt="{{assetTitle image}}">
+            </div>
+            <div class="modal__description">
+              <div class="modal__info">
+                <h5 class="modal__title">{{title}}</h5>
+                <p>{{description}}</p>
+              </div>
+              <div class="modal__data">
+                <div class="modal__account">
+                  <h6>{{bankName}}</h6>
+                  <p>{{accountType}}</br>{{accountNumber}}</p>
+                </div>
+                <div class="modal__link">
+                    {{#if vakiInfo}}
```

Comment:
Put the `if` around the `div` with the class `modal__link` so that we don't have an empty `div` when Vaki info is not provided. Like so
```
{{#if vakiInfo}}
  <div class="modal__link">
    {{{module vakiInfo}}}
  </div>
{{/if}}
```